packages/rspeedy/core/src/config/index.ts

```diff
@@ -43,6 +43,14 @@ export interface Config {
    */
   provider?: RsbuildConfig['provider']
 
+  /**
+   * Rspeedy automatically loads the .env file by default, utilizing the [Rsbuild API](https://rsbuild.dev/api/javascript-api/core#load-env-variables).
+   * You can use the environment variables defined in the .env file within your code by accessing them via `import.meta.env.FOO` or `process.env.Foo`.
+   * @see https://rsbuild.dev/guide/advanced/env-vars#env-file
+   * @defaultValue true
+   */
+  loadEnv?: boolean | undefined
```

Comment:
Rsbuild does not have this (it has the `loadEnv` option in the [`CreateRsbuildOptions`](https://rsbuild.dev/api/javascript-api/core#options)).

Maybe we could do the same thing here.

1. Add `loadEnv` to `CreateRspeedyOptions`.
2. Enable `loadEnv: true` in Rspeedy CLI (e.g.: `dev.ts`, `build.ts`, etc.)

Reply:
I see. I thought this might be a break change for current projects, so I added this flag. It gives a way to switch back to the original behavior if needed.

Reply:
This would only be a breaking change when there are `.env` files and the environment variables are used in the project without `source.define`. But it will be an error in Lynx. So I'm fine with enabling by default without opt-out.

Reply:
Rsbuild v1.3.3 adds a `--no-env` CLI flag which could opt-out `loadEnv`. See: https://github.com/web-infra-dev/rsbuild/pull/4947. We could add that, too.